apps/golden-sample-app/src/index.html

```diff
@@ -8,6 +8,33 @@
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
+    <script type="text/javascript">
+      function trackingEventsFake (eventType, event) {
+        console.log('tracking event type:', eventType, event);
+      }
+
+      // Listening for click events
+      document.addEventListener('click', function (event) {
+        let target = event.target;
+
+        if (target && target.hasAttribute('data-analytics-id')) {
+          trackingEventsFake('click', event);
+        } else {
+          while(!target.hasAttribute('data-analytics-id') && target.tagName !== 'HTML') {
```

Comment:
clicks within tagged attributes might be tricky, as you might have even multiple ids between the root element and the clicked one, but this is a nice simple way to handle it